docs/installation.rst

```diff
@@ -61,22 +61,20 @@ macOS 10.13 (High Sierra) and 10.14 (Mojave)
    macOS support is still experimental and so far
    only covers High Sierra and Mojave.
    We would love to hear your feedback in case things fail.
+   We are currently working on deploying adcc to `Anaconda <https://anaconda.org>`_.
 
 .. note::
-   Supported from adcc 0.13.2.
-
-0. **Homebrew:**
-   Support for macOS currently requires the `Homebrew <https://brew.sh>`_ package manager
-   and a recent version of ``gcc`` (e.g. ``brew install gcc@9``).
-   Hopefully, we will support ``clang`` in the future.
-   
+   The installation on macOS requires a ``clang`` compiler.
+   Make sure to have XCode and the command line tools installed.
+   ``clang`` is supported from adcc 0.13.2.
+ 
```

Comment:
I wouldn't put this full thing into a note. In fact, I think the note can now be dropped.

Reply:
No, not drop it ... just make it plain text. I think the `clang is supported from adcc 0.13.2` adds no info, because 0.13.1 was not really working anyway :smile:

Reply:
Done.